src/fragments/lib/project-setup/flutter/platform-setup/platform-setup.mdx

```diff
@@ -110,7 +110,7 @@ Add a development team in the "Signing and Capabilities" section and enable sign
 No specific configuration steps are needed to begin using the Auth category on Windows; however, depending on how your Windows configuration and the method you use to run your Flutter app, you may need to change the file path length limitation.
 
 ## Linux
-To run the Amplify-Flutter developer preview on Linux, you must install two libraries:
+Amplify-Flutter developer preview depends on libsecret and glib when targeting Linux. To run and debug an app using Amplify-Flutter, you must install the following libraries:
```

Comment:
Can we remove the dashes between Amplify Flutter please?

Reply:
@Jordan-Nelson

Reply:
@abdallahshaban557 updated 👍